packages/koa-performance/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@shopify/koa-performance",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "Creating a middleware that send performance related data through StatsD.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "scripts": {
+    "build": "tsc --p tsconfig.build.json",
+    "prepublishOnly": "yarn run build"
+  },
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "author": "Shopify Inc.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/quilt.git"
+  },
+  "bugs": {
+    "url": "https://github.com/shopify/quilt/issues"
+  },
+  "homepage": "https://github.com/Shopify/quilt/blob/master/packages/koa-performance/README.md",
+  "dependencies": {
+    "@shopify/network": "^1.4.2",
+    "@shopify/performance": "^1.2.0",
+    "@shopify/statsd": "^1.0.0",
+    "@types/koa-bodyparser": "*",
+    "@types/koa-compose": "*",
+    "change-case": "^3.1.0",
+    "koa-bodyparser": ">=4.0.0 <5.0.0",
+    "koa-compose": ">=3.0.0 <4.0.0"
+  },
+  "peer-dependencies": {
+    "koa": ">=2.0.0 <3.0.0"
+  },
+  "devDependencies": {
+    "@shopify/performance": "^1.2.0",
+    "@shopify/with-env": "^1.1.1",
+    "typescript": "~3.2.1"
```

Comment:
also realized we need to: 
1. delete the typescript dependency here
2. add a task to fix the default package generation to delete this dependency.

Reply:
done
https://github.com/Shopify/quilt/pull/1098